servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/execution/KotlinDataLoader.kt

```diff
@@ -23,6 +23,6 @@ import org.dataloader.DataLoader
  * by return type and loading values in the data fetchers.
  */
 interface KotlinDataLoader<K, V> {
```

Comment:
wondering whether it would make sense to change this to `abstract class` that extends the `DataLoader`

Reply:
DataLoader is a class, not an interface so we would have to implement one of the many constructors. I think I better option is making the `val dataLoader` into a function